### ISSUE TITLE
✨ Remember the conversation context after signing in

### ISSUE THREAD

**Issue by SofieHong** [SSWConsulting/SSW.Rules.GPT#153]
Cc: @adamcogan @pierssinclairssw 

(Checked by Piers)

Hi @jackreimers 
  
### Pain

The conversation context is removed after I sign in to RulesGPT and I have to input the same context again.

### Suggested Solution

Remember the context. (or if not possible, then give a warning)

### Tasks

- [ ] Fix it.

### Acceptance Criteria

It can remember the conversation context after sining in. (or shows a warning)

Thanks!

**Comment by jackreimers** [comment]
Duplicate of #150 which is currently blocked by #32

**Comment by jackreimers** [comment]
Resolved by #146
